supertokens_python/recipe/session/utils.py

```diff
@@ -213,6 +217,8 @@ def __init__(
     ):
         if on_token_theft_detected is None:
             on_token_theft_detected = default_token_theft_detected_callback
+        if on_try_refresh_token is None:
+            on_try_refresh_token = default_try_refresh_token_callback
```

Comment:
and tests for this to ensure that this is overridable (along with the on_clear_duplicate_session_cookies function)

Reply:
done.